core/modules/catalog/helpers/variant/checkFullConfigurationMatch.ts

```diff
@@ -0,0 +1,25 @@
+import toString from 'lodash/toString'
+import omit from 'lodash/omit'
+
+/**
+ * Checks if full coniguration matches for specific variant
+ */
+export default function checkFullConfigurationMatch (configuration, variant): boolean {
+  if (!variant || !configuration) return false
+  const configProperties = Object.keys(omit(configuration, ['price']))
+  const countMatchedConfigProperties = configProperties
+    .map(configProperty => {
+      const variantPropertyId = variant[configProperty]
+      if (configuration[configProperty] === null) {
```

Comment:
wdyt, about checking if it not exist? `if (!configuration[configProperty]) return`